language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/resolvers/parsercontext/ParserRuleStatementContextResolver.java

```diff
@@ -41,80 +40,41 @@
-            this.assignItemPriorities(prioritiesMap, completionItems);
-
-            return completionItems;
+            filteredSymbols.addAll(SymbolFilters.getFilterByClass(PackageActionFunctionAndTypesFilter.class)
+                    .filterItems(completionContext));
         } else {
-            // Fill completions if user is writing a connector init
-            List<SymbolInfo> filteredConnectorInitSuggestions = connectorInitItemFilter.filterItems(completionContext);
-            if (!filteredConnectorInitSuggestions.isEmpty()) {
-                populateCompletionItemList(filteredConnectorInitSuggestions, completionItems);
-                return completionItems;
-            }
-
-            populateCompletionItemList(completionContext.get(CompletionKeys.VISIBLE_SYMBOLS_KEY), completionItems);
-            StatementTemplateFilter statementTemplateFilter = new StatementTemplateFilter();
-            // Add the statement templates
-            completionItems.addAll(statementTemplateFilter.filterItems(completionContext));
+            filteredSymbols.addAll(SymbolFilters.getFilterByClass(ConnectorInitExpressionItemFilter.class)
+                    .filterItems(completionContext));
+            filteredSymbols.addAll(completionContext.get(CompletionKeys.VISIBLE_SYMBOLS_KEY));
+            completionItems.addAll(SymbolFilters.getFilterByClass(StatementTemplateFilter.class)
+                    .filterItems(completionContext));
 
             CompletionItem xmlns = new CompletionItem();
             xmlns.setLabel(ItemResolverConstants.XMLNS);
             xmlns.setInsertText(Snippet.NAMESPACE_DECLARATION.toString());
             xmlns.setInsertTextFormat(InsertTextFormat.Snippet);
             xmlns.setDetail(ItemResolverConstants.SNIPPET_TYPE);
-            xmlns.setSortText(Priority.PRIORITY7.name());
             completionItems.add(xmlns);
 
-            CompletionItem workerItem = new CompletionItem();
-            workerItem.setLabel(ItemResolverConstants.WORKER);
-            workerItem.setInsertText(Snippet.WORKER.toString());
-            workerItem.setInsertTextFormat(InsertTextFormat.Snippet);
-            workerItem.setDetail(ItemResolverConstants.WORKER_TYPE);
-            workerItem.setSortText(Priority.PRIORITY6.name());
-            completionItems.add(workerItem);
-
-            CompletionItem xmlAttribute = new CompletionItem();
-            xmlAttribute.setInsertText(Snippet.XML_ATTRIBUTE_REFERENCE.toString());
-            xmlAttribute.setInsertTextFormat(InsertTextFormat.Snippet);
-            xmlAttribute.setLabel("@");
-            xmlAttribute.setDetail("xmlAttribute");
-            xmlAttribute.setSortText(Priority.PRIORITY6.name());
-            completionItems.add(xmlAttribute);
-
-            // Add the var keyword
             CompletionItem varKeyword = new CompletionItem();
             varKeyword.setInsertText("var ");
             varKeyword.setLabel("var");
```

Comment:
can we keep "var" as a constant as it is used in few places? this is just a suggestion.

Reply:
Added a constant as per suggestion